src/signals/incident-management/containers/IncidentSplitContainer/IncidentSplitSelectInput.js

```diff
@@ -0,0 +1,56 @@
+import React, { useCallback, useState } from 'react';
```

Comment:
This is a component and can be placed in the `components` subfolder.

Reply:
I don't think it's flexible enough yet to live there..

Reply:
What I meant is a new folder `src/signals/incident-management/containers/IncidentSplitContainer/components`, Perhaps was that not clear enough.